components/sensor/atm90e32.rst

```diff
@@ -0,0 +1,139 @@
+ATM90E32 Power Sensor
+====================
+
+.. seo::
+    :description: Instructions for setting up ATM90E32 energy metering sensors
+    :image: atm90e32.png
+    :keywords: ATM90E32, CircuitSetup, Split Single Phase Real Time Whole House Energy Meter, Expandable 6 Channel ESP32 Energy Meter Main Board
+
+The ``atm90e32`` sensor platform allows you to use your ATM90E32 voltage/current and power sensors
+(`datasheet <http://ww1.microchip.com/downloads/en/devicedoc/Atmel-46003-SE-M90E32AS-Datasheet.pdf>`__) sensors with
+ESPHome. This sensor is commonly found in CircuitSetup 2 and 6 channel energy meters.
+
+The atm90e32 IC can measure up to three AC voltages although typically only one
+voltage measurement would be used for the mains electricity phase of a
+household. Three current measurements are read via CT clamps. 
+
+The CircuitSetup Split Single Phase Real Time Whole House Energy Meter can read 2 current channels and one voltage channel.
```

Comment:
`CircuitSetup Split Single Phase Real Time Whole House Energy Meter` that's mouthful :)

Could we shorten this, make it a link or put quotes around it?